src/Text/Text.spec.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { mount, render } from 'enzyme';
```

Comment:
@cameron-jones-volusion Looks like `mount` is unused. I would actually suggest replacing the render function with `mount(...).render()`